state/pool.go

```diff
@@ -424,7 +424,7 @@ func (p *StatePool) Close() error {
 		p.txnWatcherSession.Close()
 	}
 	p.mu.Unlock()
-	// As with above and the other watchers. Unlock while releas
+	// As with above and the other watchers.
```

Comment:
"As with above and the other watchers, unlock while releasing the state session."